config/recipes/traefik/02-elastic-stack.yaml

```
@@ -41,6 +41,10 @@ metadata:
   labels:
     app: hulk
 spec:
+  config:
+    xpack.fleet.packages:
+    - name: apm
+      version: latest
   version: 8.4.2
   count: 1
```

How about keeping th `version`, `count`, `config` fields in the same order:
```suggestion
  version: 8.4.2
  count: 1
  config:
    xpack.fleet.packages:
    - name: apm
      version: latest
```